exercises/concept/lasagna/.docs/hints.md

```diff
@@ -26,7 +26,7 @@
 - You can use the [mathematical operator for addition][operators] to add values.
 
 [methods]: https://launchschool.com/books/ruby/read/methods
-[return]: https://www.freecodecamp.org/news/idiomatic-ruby-writing-beautiful-code-6845c830c664/
+[return]: https://www.freecodecamp.org/news/idiomatic-ruby-writing-beautiful-code-6845c830c664/#implicit-return
```

Comment:
This is a long article and the relevant part is far away from the beginning, so I thought it's better to link right to it.